src/components/NavigationDrawer.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { StyleSheet, View, Image, TouchableOpacity } from 'react-native'
+import BottomSheet from 'reanimated-bottom-sheet'
+import Icon from 'react-native-vector-icons/FontAwesome5'
+
+import logo from '../../assets/logo.png'
+
+type NavigationBarProps = {
+    navigation: StackNavigationProp<>;
+  }
+
+const styles = StyleSheet.create( {
+  navigationPanel: {
+    height: '100%',
+    backgroundColor: '#f9f9fa',
+    opacity: 0.92,
+  },
+  navigationPanelHeader: {
+    backgroundColor: '#f9f9fa',
+    paddingTop: 7,
+    borderTopLeftRadius: 20,
+    borderTopRightRadius: 20,
+    alignItems: 'center',
+    opacity: 0.92,
+  },
+  navigationPanelHandle: {
+    width: 80,
+    height: 2,
+    borderRadius: 5,
+    backgroundColor: '#00000040',
+  },
+  navigationPanelIconContainer: {
+    flexDirection: 'row',
+    paddingTop: '5%',
+    paddingBottom: '10%',
+  },
+  navigationIconSpacing: {
+    paddingRight: 30,
+    paddingLeft: 30,
+  },
+  navigationPanelButtonContainer: {
+    flexDirection: 'row',
+    justifyContent: 'space-around',
+  },
+} )
+
+const Buttons = [
+  [ 'clock', 'WIP' ],
+  [ 'bookmark', 'WIP' ],
+  [ 'search', 'Search' ],
+  [ 'square', 'WIP' ],
+  [ 'ellipsis-h', 'WIP' ],
+]
+
+const NavigationBar: React.FC<NavigationBarProps> = ( { navigation } ) => {
+  const renderNavigationHeader = () => (
+    <View style={styles.navigationPanelHeader}>
+      <View style={styles.navigationPanelHandle} />
+
```

Comment:
can we try and clear up the whitespace between the elements please :)

Reply:
@swissarmykirpan I actually preferred the white space for readability, kinda told him to add it loool. Having it tightly stacked together (either code and JSX) makes it a little harder to scan “related” stuff